### DISCUSSION TITLE
Webhook conditions

### DISCUSSION THREAD

**Discussion by AndMrzv** [netbox-community/netbox#8886] [Q&A] [answered]
Recently updated from 3.0 to 3.1.9 and wanted to try new conditional webhooks but it seems that if I put any check at all in "Conditions" box the webhook is not triggering, POST never comes.
For example I added a webhook for create and delete events - it works. Now I add a check to filter for deletions and messages stop completely. Tried adding "op":"eq", tried replacing and with or, tried filtering for username and other attributes - same thing.
Do conditions actually work? Any way to diagnose this if it is just me?
![Снимок экрана от 2022-03-16 17-11-38](https://user-images.githubusercontent.com/99031446/158610027-40e40af0-c738-4c7d-9596-f72b3472f034.png)

**Comment by jeremystretch** [comment, marked as answer]
The conditions apply to the object which triggers the webhook, not the webhook itself. `attr` should be an attribute of the object.

**Reply by AndMrzv** [reply]
Ok, I didn't realise from the https://demo.netbox.dev/static/docs/reference/conditions/ page that conditions apply to the {"data": {}} block and tried to adress it as a nested. Now I got it working.
Thank you!

**Reply by dietybright** [reply]
Hi,

Anyway to set condition to the webhook inself? I want to trigger webhook only if any diff on the change by using prechange, postchange value
